securedrop/tests/test_encryption.py

```diff
@@ -138,9 +138,9 @@ def test_encrypt_source_file(self, config, tmp_path):
                 encrypted_file_path_out=encrypted_file_path,
             )
 
-        # And the output file contains the encrypted data
+        # And the output file doesn't contain the file plaintext
         encrypted_file = encrypted_file_path.read_bytes()
-        assert encrypted_file.startswith(b"-----BEGIN PGP MESSAGE-----")
+        assert file_to_encrypt_path.read_bytes() not in encrypted_file
```

Comment:
I raised an eyebrow at this, since `file_to_encrypt_path.read_bytes()` could be up to 500 MB: couldn't we just check the first chunk, of whatever size?  But of course this is just a test, so we control the plaintext input as well as the ciphertext output.